docs/src/index.md

```diff
@@ -2,10 +2,6 @@
 layout: home
 ---
 
-> **Slinkity is now available as an early alpha!** No, it's not ready for production use. But if you want to have some fun and don't mind logging bugs, head to the [quick start guide](/docs/quick-start/) :)
-
-{% include 'home/heading-links.html' %}
-
```

Comment:
Moved these sections into `_layouts/home`. Ensures this is contained in the `home__heading` container for clearer structure!